CodeEdit/Utils/WorkspaceClient/Model/FileIcon.swift

```diff
@@ -101,24 +225,36 @@ enum FileIcon {
     /// If not specified otherwise this will return `Color.accentColor`
     static func iconColor(fileType: FileType) -> Color {
         switch fileType {
-        case .swift, .html:
+        case .julia, .perl, .app, .mov, .mp4:
+            return .gray
+        case .swift, .html, .sql, .htm:
             return .orange
-        case .java:
+        case .java, .svelte, .scala, .ruby, .d:
             return .red
         case .js, .entitlements, .json, .LICENSE:
             return Color("SidebarYellow")
-        case .css, .ts, .jsx, .md, .py:
+        case .c, .cpp, .css, .ts, .jsx, .r, .md, .py, .lua, .cobol, .cc, .groovy, .gvy, .gy, .gsh, .xml:
             return .blue
-        case .sh:
+        case .cs, .sh, .lz:
             return .green
+        case .doc, .docm, .docx:
+            return Color(red: 0.168, green: 0.336, blue: 0.690, opacity: 1.0)
```

Comment:
One thing we could do for all the `Color(red:green:blue:opacity)` cases in order to make it more descriptive and readable, is to create color assets similar as `SidebarYellow` in line 235

Reply:
Would we put these cases in `FileIcon.swift` or another file?